CustomerInformation/CBCustomerInfo.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  CBCustomerInfo.swift
+//  Chargebee
+//
+//  Created by ramesh_g on 06/02/23.
+//
+
+import Foundation
+
+
+public struct CBCustomerInfo {
```

Comment:
`CBCustomer` shoulds better to me. WDYT?

Reply:
Also this should hold all the customer relevant fields, including the `customerId`

Reply:
Updated all details in CBCustomer object.